src/math/Color.js

```diff
@@ -347,7 +347,7 @@ class Color {
 
 		ColorManagement.fromWorkingColorSpace( _color.copy( this ), colorSpace );
 
-		return clamp( _color.r * 255, 0, 255 ) << 16 ^ clamp( _color.g * 255, 0, 255 ) << 8 ^ clamp( _color.b * 255, 0, 255 ) << 0;
+		return Math.round( clamp( _color.r * 255, 0, 255 ) ) << 16 + Math.round( clamp( _color.g * 255, 0, 255 ) ) << 8 + Math.round( clamp( _color.b * 255, 0, 255 ) );
```

Comment:
The problem was that when floats are internally converted in JavaScript to integers for bitwise shifts, they are, I think, rounded down -- now they are rounded to the nearest integer (maybe we can introduce a separate function like `integerClamp`?).